### DISCUSSION TITLE
existingSecretAdminPassword is not working

### DISCUSSION THREAD

**Discussion by hashanmp** [goharbor/harbor-helm#1428] [Q&A]
Hello!
I am using helm chart version 1.11.1 to deploy harbor on our EKS cluster using argoCD but for some reason even when I use existingSecretAdminPassword parameter it won't pick up the Kubernetes secret for admin user. 
also I am using `existingSecret` for external database it works without an issue.
can someone help me here? 🙏 

here is my value file looks like
```
harbor:
  ---
  existingSecretAdminPassword: "harbor-creds"
  existingSecretAdminPasswordKey: "HARBOR_ADMIN_PASSWORD"
  --
```

**Comment by Deahan-We** [comment]
existingSecretAdminPassword values is not tagged yet.

It is just in Master branch

**Reply by hashanmp** [reply]
Thank you @Deahan-We for the update. 
do you happen to know the when will feature rollout?
